Nanoleaf.Client/Nanoleaf.Client/Models/Responses/Layout.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.ComponentModel;
+using Newtonsoft.Json;
+
+namespace Nanoleaf.Client.Models.Responses {
+	/// <summary>
+	/// Layout response returned from a layout request
+	/// </summary>
+	[Serializable]
+	public class Layout {
```

Comment:
Let's follow the existing code design. Curly braces come on the next line.

Reply:
Fixed with 21f16ef.